Assignment6.Rmd

```diff
@@ -77,7 +76,30 @@ table(M2$certified, M2$predict2)
 Choose a data file from the (University of Michigan Open Data Set)[https://github.com/bkoester/PLA/tree/master/data]. Choose an outcome variable that you would like to predict. Build two models that predict that outcome from the other variables. The first model should use raw variables, the second should feature select or feature extract variables from the data. Which model is better according to the cross validation metrics?
 
 ```{r}
+library(dplyr)
+library(rpart)
+t <- read.csv("student.course.csv", header = TRUE)
+t1 <- filter(t, TERM == 135)
+t1 <- t1[c(2001:4000),]
+
+c.tree1 <- rpart(SUBJECT ~ DIV+GPAO,  method="class", data=t1)
+c.tree1 <- rpart(SUBJECT ~ GPAO + CATALOG_NBR,  method="class", data=t1)
+c.tree1 <- rpart(SUBJECT ~ GRD_PTS_PER_UNIT + CATALOG_NBR,  method="class", data=t1)
+
+c.tree1 <- rpart(SUBJECT ~ GPAO + GRD_PTS_PER_UNIT + DIV + CATALOG_NBR, method = "class", data = t1)
```

Comment:
Try to build a model that feature extracts variables from the data. One way to do this is PCA.